pkg/common/vclib/connection.go

```diff
@@ -181,12 +181,6 @@ func (connection *VSphereConnection) NewClient(ctx context.Context) (*vim25.Clie
 	if err != nil {
 		return nil, err
 	}
-	if klog.V(3) {
-		s, err := session.NewManager(client).UserSession(ctx)
```

Comment:
Is it safe to actually remove this call, I was just meant the log line previously but if `s` is not used anywhere this seems fine.

Reply:
yep, it's completely safe to remove. if anything, I don't think we should be printing the session key in the logs anyways. it could be used to hijack a VC session.

Reply:
verified that CPI behaves correctly removing these lines